src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
+  , Label
+  , Package
+  , Config
+  ) where
+
+data Label = Label
+  { name        :: String
```

Comment:
We are using 4-spaces indentation rule by our style guide.

Reply:
Also, @kahlil29 is right, it's better to use `Text` fields here and further :+1:

Reply:
Fields names should be started with the data type name prefix:
```suggestion
  { labelName        :: !Text
```